### PR TITLE
feat(asset): merge asset & asset-calibration plugins

### PR DESCRIPTION
**Pull request by Catalin-web**
# Pull Request

## 🤨 Rationale

Task: [Move calibration forecast plugin](https://ni.visualstudio.com/DevCentral/_backlogs/backlog/ASW%20SystemLink%20Asset%20Mgmt/Features/?workitem=2892130)

## 👩‍💻 Implementation

Merge the 2 plugins.
Remove the asset-calibration plugin, and just keep the asset one

## 🧪 Testing

Moved test, modified tests+
merged plugins

## ✅ Checklist

- [x] This PR has a title that follows the [commit message format](https://github.com/ni/systemlink-grafana-plugins#commit-message-format).

### CODE REVIEW

**Review by cameronwaterman — APPROVED** [review]
Just reviewing changes to codeowners change - please wait for other review before merging.

File: src/datasources/asset/components/editors/asset-summary/AssetSummaryDataSource.ts
Comment (mure) [inline src/datasources/asset/components/editors/asset-summary/AssetSummaryDataSource.ts:1]: My expectation is that things in the `components` directory are React components. We should move the data sources into their own folder or just put them in the root (`/asset`).

**Review by mure — APPROVED** [review]

**Review by kkerezsi — APPROVED** [review]

**Pull request merged**
